src/test/integration/users-test.js

```diff
@@ -50,3 +55,50 @@ it('logs in as a user', async () => {
   const altInfo = await userInfo(server, createJWT);
   expect(altInfo.email).toBe(TEST_USER.email);
 });
+
+it('edits an invalid user', async () => {
+  const jwt = await createUser(server, TEST_USER);
+  const rsp = await pFinal(
+    editUser(server, jwt, Object.assign(TEST_USER_UPDATE_1, {id: 0})));
+  expect(rsp.response.status).toEqual(404);
+});
+
+it('edits a user', async () => {
+  const jwt = await createUser(server, TEST_USER);
+  const user = await Users.getUser(TEST_USER.email);
```

Comment:
I'd prefer for this to use the existing `userInfo(server, jwt)` function to get the user description rather than retrieving it directly from the model.

Reply:
Forgot I added the ID to the info returned by the API. Done.